core/src/main/java/org/bptlab/cepta/operators/SumOfDelayAtStationFunction.java

```diff
@@ -0,0 +1,83 @@
+package org.bptlab.cepta.operators;
+
+import java.lang.Object;
+import java.util.HashMap;
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.common.functions.FilterFunction;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.api.java.tuple.Tuple2;
+
+import org.bptlab.cepta.models.events.train.TrainDelayNotificationProtos.TrainDelayNotification;
+
+import org.apache.flink.util.Collector;
+
+import org.bptlab.cepta.utils.triggers.CustomCountTrigger;
+import org.apache.flink.streaming.api.functions.windowing.ProcessWindowFunction;
+import org.apache.flink.streaming.api.windowing.windows.GlobalWindow;
+import org.apache.flink.streaming.api.windowing.assigners.GlobalWindows;
+
+import org.apache.flink.api.java.functions.KeySelector;
+
+public class SumOfDelayAtStationFunction {
+
```

Comment:
Please add a comment on which type of stream this works and what it does, as well as an indication what is in the resulting stream